R/pkg/R/serialize.R

```diff
@@ -136,8 +136,17 @@ serializeRow <- function(row) {
 }
 
 writeRaw <- function(con, batch) {
+  dataSize <- length(batch)
   writeInt(con, length(batch))
-  writeBin(batch, con, endian = "big")
+
+  LENTMAX <- 2147483000 # Slightly less than R_LEN_T_MAX macro in R
```

Comment:
`R_LEN_T_MAX` is not readable in R. I am going to make it a configurable option. This will give users/admins control it as well. I will probably add another config to set a maximum value (that depending on the config can be greater than or smaller than `R_LEN_T_MAX`)